wowchemy/layouts/slides/baseof.html

```diff
@@ -20,55 +21,52 @@
 
   <title>{{ .Title }} | {{ site.Title }}</title>
 
-  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/css/reveal.min.css">
+  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/reveal.min.css">
   {{- $theme := $.Param "slides.theme" | default "black" -}}
-  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/css/theme/{{ $theme }}.min.css">
+  <link rel="stylesheet" href="{{ $cdn_url_reveal }}/theme/{{ $theme }}.min.css" id="theme" />
```

Comment:
If this ID is needed, consider naming it `reveal-theme` so it's clear which theme it's referring to elsewhere.

Reply:
needed by reveal menu plugin to identify the available list of themes. Cant change it im afraid

Reply:
Consider requesting the author to add support for custom identifiers (or helping contribute to the library), otherwise this is open to conflicts (especially if it ends up embedded in other pages (https://revealjs.com/presentation-size/#embedded ), and may even conflict with multiple embedded slides on a page?) as well as causing confusion for future maintainers/contributors given the other themes in use.

Reply:
removed it .. website authors need to mess with configuration in the frontend to get theme selection to work 
I think its still configurable so we are not losing much